### PR TITLE
Fixes #351 - Update readme

### LINKED ISSUE #351: Update Readme
(after version bump)
* Foreman dependencies
* installation instructions
* year

### PR DESCRIPTION
**Pull request by nadjaheitmann**
Fixes #351

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,8 @@ Some features will remain in core:
 
 |Foreman version|Plugin version|Notes                                     |
 |---------------|--------------|------------------------------------------|
-| >= 3.5        | ~> 5.0       | Required                                 |
+| >= 3.7        | ~> 6.0       | Required                                 |
+| ~> 3.5, 3.6   | ~> 5.0       | Required                                 |
```
Comment (nadjaheitmann) [inline README.md:37]: Can we do better than this?

File: README.md [hunk 2]
```diff
@@ -77,7 +78,7 @@ Fork and send a Pull Request. Thanks!
 
 ## Copyright
 
-Copyright (c) *2022* *The Foreman developers*
+Copyright (c) *2023* *The Foreman developers*
```
Comment (ekohl) [inline README.md:81]: It's more common to make it 2022-2023.

File: README.md (hunk 1 above)
Comment (ekohl) [reply README.md:37]: I think it's good enough.

File: README.md (hunk 2 above)
Comment (nadjaheitmann) [reply README.md]: I changed it, I think that makes sense. I found it not being very consistent in the Foreman world, though.

File: README.md [hunk 3]
```diff
@@ -77,7 +78,7 @@ Fork and send a Pull Request. Thanks!
 
 ## Copyright
 
-Copyright (c) *2022* *The Foreman developers*
+Copyright (c) *2020-2023* *The Foreman developers*
```
Comment (ekohl) [inline README.md:81]: Note to self: first commit is from 2020, so backdating this is correct.

**Review by ekohl — APPROVED** [review]
@stejskalleos since you self assigned this I'm not merging this yet, but it looks good to me.

**Review by stejskalleos — APPROVED** [review]
:green_apple: **LGTM**

**Comment by stejskalleos** [comment]
Thanks @nadjaheitmann @ekohl

**Pull request merged**